### PR TITLE
Feature/text 2016

### PR DESCRIPTION
**Pull request by mistermark**

### CODE REVIEW

File: app/content/index.json [hunk 1]
```diff
@@ -3,26 +3,32 @@
     "siteurl": "http://hackathon.backbase.com"
   },
   "hero": {
-    "h1": "Hack the system & win awesome prizes!",
-    "p": "Welcome to the 2015 Backbase Hackathon in Amsterdam and online, a two-day chance to network, learn and share knowledge with your peers next <b>19th and 20th of June 2015.</b>",
+    "h1": "Rewards for disrupting the banking system!",
```
Comment (jelmerdejong) [inline app/content/index.json]: Can we change that to something like "Create the bank of the future!" or "Deliver the next generation of banking experiences"?

File: app/partials/apis.hbs [hunk 2]
```diff
@@ -1,13 +1,36 @@
...
-  <h2>{{html apis.h2}}</h2>
-  <p>{{html apis.subtitle}}</p>
-  {{#each apis.ideas}}
-      <h4>{{html title}}</h4>
-      <ul>
-          <li>{{html description}}</li>
-      </ul>
-  {{/each}}
-  <p>{{html apis.footer}}</p>
+  <h2>APIs you might want to use</h2>
```
Comment (jelmerdejong) [inline app/partials/apis.hbs]: Can we add Stripe? As payment API probably something easy and quick to build upon.

File: app/partials/goals.hbs [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
+  <h3>Compete in one of these categories:</h3>
+
+  <h4>Freestyle: Hack the banking industry</h4>
+  <p>The freestyle category lets you run wild with your imagination and skills, with basic help if required. The goal is to disrupt the banking industry. To help you with this, you can use our Breeze API or any standalone model (if needed).</p>
+  <p>About Breeze API: URL to Breeze server: Documentation of Breeze API can be found <a href="http://hackathon.backbase.dev:8080/portalserver/home">here</a>. (to be updated!)</p>
+  <p>Tags: Banking/Freestyle</p>
+  <p><em>Last year, the team Exodus 5:7 had 3D-printed a piggy bank that counts coins as they fall in. This information is registered and sent to the online banking platform, which then displays a real-time actual balance. Together with this, the team created a special (and extremely simplified) interface designed for kids, to begin understanding simple concepts such as having to wait to get something you want.</em></p>
+
+  <h4>Build on top of a Backbase product</h4>
```
Comment (jelmerdejong) [inline app/partials/goals.hbs]: Build on top of the Backbase Platform

File: app/partials/goals.hbs [hunk 4]
```diff
@@ -1,14 +1,24 @@
...
+  <p>In this category, the aim is to improve Backbase products by getting under their skin and hacking them! There are plenty of products to tackle, such as CXP, Forms, Launchpad, plus any Backbase internal tools. This is a chance to walk the talk and enhance the Backbase product experience for everyone!</p>
```
Comment (jelmerdejong) [inline app/partials/goals.hbs]: In this category, the aim is to improve the Backbase Platform by getting under their skin and hacking them! There are plenty of areas to tackle, such as CX Services, our Mobile or Web SDK, the Widget Collections, Manager, Forms, plus all Backbase internal (dev) tools. This is a chance to walk the talk and enhance the Backbase product experience for everyone!

File: app/partials/goals.hbs [hunk 5]
```diff
@@ -1,14 +1,24 @@
...
+  <p>Tags: Backbase Products/Add-ons/Improvements</p>
```
Comment (jelmerdejong) [inline app/partials/goals.hbs]: Backbase Add-ons & Improvements

File: app/partials/inspiration.hbs [hunk 6]
```diff
@@ -1,13 +1,24 @@
...
+  <p><a href="http://blog.backbase.com/4344/backbase-2015-hackathon/">See the examples from 2015</a></p>
+
+  <h4>Stock Tracker</h4>
+  <ul>
+      <li>Use the Backbase Mobile SDK to create a new stock tracker app that visualizes how people look at their portfolio in a completely different way.</li>
+  </ul>
+
+  <h4>Analytics Power!</h4>
+  <ul>
+      <li>Add a new Analytics App to CXP Manager 5.6 that consumes the most needed reports from Google Analytics, in real time.</li>
```
Comment (jelmerdejong) [inline app/partials/inspiration.hbs]: Manager, without CXP

File: app/partials/inspiration.hbs [hunk 7]
```diff
@@ -1,13 +1,24 @@
...
+  </ul>
+
+  <h4>Bank 10.0</h4>
+  <ul>
+      <li>Visualize how the Bank of The Future would look like and create a Launchpad Theme to make it real. Inspiration?</li>
```
Comment (jelmerdejong) [inline app/partials/inspiration.hbs]: dich Launchpad, just theme

File: app/partials/joinus.hbs [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<div class="col-md-2"></div>
+<div class="col-md-8">
+  <h2>Join us on this 2-day hackathon in Amsterdam!</h2>
+
+  <p>The 2nd Edition of the Backbase Hackathon invites you to apply today. Don’t waste time! Get this form filled in as soon as possible! The aim is to compete, so we can innovate together and move digital banking forwards. Here are some simple rules to abide by:</p>
+  <ol>
+    <li>Fresh hacking - This means you must present a brand new product or feature that didn’t exist before. Your hack must be crafted entirely during the hackathon.</li>
+    <li>Work from a Backbase location – You can join the Hackathon remotely from any Backbase location. Please bring your own laptop.</li>
+    <li>Sign up as a team– A team can consist of 1-5 members, and can comprise a mix of talents. For instance, a team could include a coder, a user experience expert, or an entrepreneur – any flavor will do.</li>
```
Comment (jelmerdejong) [inline app/partials/joinus.hbs]: add space before  -

File: app/partials/joinus.hbs [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+  </ol>
+  <p>At the end of the Hackathon, each team leader will present a 15-minute prototype demo to a jury of Backbase managers, as well as representatives of the Hackathon sponsor company.</p>
```
Comment (jelmerdejong) [inline app/partials/joinus.hbs]: we settled for 7 minutes right?

File: app/content/index.json (hunk 1 above)
Comment (mistermark) [inline app/content/index.json]: Can you pick one? I'll change the one you pick.

File: app/partials/apis.hbs (hunk 2 above)
Comment (mistermark) [inline app/partials/apis.hbs]: @fer Can you add information about this on the document? Let me know when you did.

**Comment by mistermark** [comment]
Closing this one. Creating a new one.

**Pull request closed without merging**